amida-tech/indaba-client
# INBA-266/ Cancel button for Create New Project modal.

#### What's this PR do?
It all started with a simple request to add a "cancel" button to the "Create New Project" wizard. Which in turn kind of grew because that modal was relying on the React state instead of using a reducer. So, I slipped it into the navReducer which was pre-existing. Action and action type later, and I had to take a look at the Modal.js. Realized I should probably break that up into two components and that was a mess... but hey, it's all fixed now I guess. 


#### Related JIRA tickets:
INBA-266 and its epic INBA-259.

#### How should this be manually tested?
Go to `http://localhost:3000/project/101` and click on the "Create" button.
Click cancel. Ensure that it cancels.
Click on the "Create" button again.
Click "Create New Project" and ensure you go to the screen.

... then go back and test every modal in existence. 

#### Any background context you want to provide?

#### Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4026454/28801485-c4975fca-761f-11e7-8bfd-ce01c971ba5a.png)
